contrib/exporters/secadvisor/mod/transform.go

```diff
@@ -38,7 +38,12 @@ func NewTransform(cfg *viper.Viper) (interface{}, error) {
 		return nil, err
 	}
 
-	resolver, err = NewResolveMulti(resolver.(Resolver))
+	resolver2, err := NewResolveVm(cfg)
```

Comment:
you have not added NewResolveVm() to this commit ... so this will not build

Reply:
done